decidim-elections/app/commands/decidim/elections/admin/setup_election.rb

```diff
@@ -44,6 +44,10 @@ def answers
           Decidim::Elections::Answer.where(question: questions)
         end
 
+        def ballot_styles
+          @ballot_styles ||= questions.map(&:ballot_styles).flatten.uniq
```

Comment:
💬 (Read this after the comment in the `Question` model)
It seems that this is the only place where you use the questions -> ballot_styles relationship. Maybe you could create the hash of ballot styles with something like this, and remove that association and the `ballot_styles_data` method:

``ruby
all_voting_ballot_styles.map do |ballot_style|
  questions = ballot_style.questions_for(elections)
  [ballot_style.slug, questions] if questions.any?
end.compact.to_h
```

Reply:
As discussed offline, removed the `has_many` relationship and, for the moment being, attempting accesing the ballot styles directly from the command https://github.com/decidim/decidim/pull/7822/commits/04863cd0921cb8a5f337a027bac77794114bacbd